docs/source/sed/sed_example.ipynb

```diff
@@ -509,13 +509,47 @@
    "source": [
     "sed2 = Sed(sed.lam, np.array([sed.lnu, sed.lnu * 2]) * erg /s /Hz)\n",
     "\n",
-    "sed2.measure_window_lnu((1400, 1600) * Angstrom)\n",
-    "sed2.measure_beta()\n",
-    "sed2.measure_beta(window=(1250, 1750, 2250, 2750)* Angstrom)\n",
-    "sed2.measure_balmer_break()\n",
-    "sed2.measure_index(\n",
+    "print(sed2.measure_window_lnu((1400, 1600) * Angstrom))\n",
+    "print(sed2.measure_beta())\n",
+    "print(sed2.measure_beta(window=(1250, 1750, 2250, 2750)* Angstrom))\n",
+    "print(sed2.measure_balmer_break())\n",
+    "print(sed2.measure_index(\n",
     "    (1500, 1600) * Angstrom, (1400, 1500) * Angstrom, (1600, 1700) * Angstrom\n",
-    ")"
+    "))\n",
+    "print(sed2.calculate_ionising_photon_production_rate(\n",
+    "    ionisation_energy=13.6 * eV, limit=1000\n",
+    "))"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "91953298",
+   "metadata": {},
+   "source": [
+    "This even works for N-dimensional spectra. For example, if we convert an entire grid into a set of spectra:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "0cbf660f",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "sed3 = grid.get_sed(spectra_type='incident')\n",
+    "print(sed3.lnu.shape)\n",
+    "\n",
+    "print(sed3.measure_window_lnu((1400, 1600) * Angstrom))\n",
+    "# print(sed3.measure_beta())  # DOES NOT WORK\n",
```

Comment:
Can't merge stuff like this into main for it to appear in the docs. Please remove it or fix it.

Reply:
Done